### PR TITLE
Mod 6 ober page

### PR DESCRIPTION
**Pull request by rmzhen**

### CODE REVIEW

**Review by LokiTheShadySpirit — APPROVED** [review]
Yippie I love sorting.

**Review by LokiTheShadySpirit — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -75,19 +62,15 @@ function App() {
...
+        if (oldIndex < index) {
           order.dates[index] = new Date();
-          localStorage.setItem(order.id, JSON.stringify(order.dates));
         }
 
         const orderToUpdate = unfilteredOrders.find((order) => order.id === id);
         orderToUpdate.status = statuses.indexOf(status);
+        orderToUpdate.statusTimes[index] = order.dates[index];
         changeOrderStatus(orderToUpdate);
+        console.log(orderToUpdate);
```
Comment (judahlit) [inline src/App.js]: console.log should be removed

File: src/pages/KitchenPage.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <Column
+          index={column.statusIndex}
+          statuses={props.statuses}
+          key={column.status}
+          text={column.status}
+          orders={column.orders}
+          changeStatus={props.changeStatus}
+        />
+      ))}
+      console.log("hello world")
```
Comment (judahlit) [inline src/pages/KitchenPage.js]: console.log should be removed

File: src/pages/WaiterPage.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import Column from "../components/column";
+import Order from "../components/order";
+import "../styling/waiterPage.css";
+export default function WaiterPage(props) {
+  useEffect(() => {
+    console.log(props.columns);
```
Comment (judahlit) [inline src/pages/WaiterPage.js]: console.log should be removed

File: src/services/OrderWebSocketService.js [hunk 4]
```diff
@@ -1,35 +1,32 @@
...
+  onNewOrder = (data) => {
+    return;
+  };
+
+  onUpdateOrder = (data) => {
+    return;
+  };
+
+  updateOrder(order) {
+    console.log(order);
```
Comment (judahlit) [inline src/services/OrderWebSocketService.js]: console.log should be removed

**Review by judahlit — CHANGES_REQUESTED** [review]

File: src/pages/KitchenPage.js (hunk 2 above)
Comment (rmzhen) [reply src/pages/KitchenPage.js]: Fixed

File: src/pages/WaiterPage.js (hunk 3 above)
Comment (rmzhen) [reply src/pages/WaiterPage.js]: Fixed

File: src/services/OrderWebSocketService.js (hunk 4 above)
Comment (rmzhen) [reply src/services/OrderWebSocketService.js]: Fixed

**Review by Luxahh — APPROVED** [review]

**Review by judahlit — APPROVED** [review]

**Pull request merged**